docs/modules/getting-started/pages/security.adoc

```diff
@@ -0,0 +1,190 @@
+
+To detect if Developer Mode has been enabled on the device the `SecurityChecks#NO_DEVELOPER_MODE` check can be used.
+This check uses Android's link:https://developer.xamarin.com/api/type/Android.Provider.Settings+Global/[Settings.Global] class.
+
+==== Debugger Detection (NO_DEBUGGER)
+
+To detect if the application is running in Debug Mode, `SecurityChecks#NO_DEBUGGER` function can be used.
+
+[NOTE]
+====
+The check uses different approaches on Android and iOS to detect if the app is running into an emulator
+[horizontal]
+ANDROID:: Uses the Android's link:https://developer.xamarin.com/api/type/Android.OS.Debug/[Debug] class.
+iOS:: Uses the iOS link:https://docs.microsoft.com/en-us/dotnet/api/System.Diagnostics.Debugger/[Debugger] class
+====
+
+==== Emulator Detection (NOT_IN_EMULATOR)
+
+To detect if the app is being run on an emulator the `SecurityChecks#NOT_IN_EMULATOR` function can be used.
+
+[NOTE]
+====
+The check uses different approaches on Android and iOS to detect if the app is running into an emulator
+[horizontal]
+ANDROID:: Uses the Android's link:https://developer.xamarin.com/api/type/Android.OS.Build/[Build] class.
+iOS:: Uses the iOS link:https://developer.xamarin.com/api/field/ObjCRuntime.Runtime.Arch/[Arch] class
+====
+
+==== Device Lock Detection (SCREEN_LOCK_ENABLED)
+
+To detect if a device has a lock screen set (with pin, fingerprint, pattern) the `SecurityChecks#SCREEN_LOCK_ENABLED` function can be used.
+
+[NOTE]
+====
+The check uses different approaches on Android and iOS to detect if the app is running into an emulator
+[horizontal]
+ANDROID:: Uses the Android's link:https://developer.xamarin.com/api/type/Android.App.KeyguardManager[KeyguardManager] class.
+iOS:: Uses the iOS link:https://developer.xamarin.com/api/type/MonoTouch.LocalAuthentication.LAContext/[LAContext] class
+====
+
+==== App Data Backup Detection (ALLOW_BACKUP_DISABLED)
+
+IMPORTANT: This check is available for Android only
+
+To detect whether the application’s data is configured to be synchronized across devices the `SecurityChecks#ALLOW_BACKUP_DISABLED` function can be used.
+The link:https://developer.android.com/guide/topics/manifest/application-element[allowBackup] flag determines whether to allow the application to
+participate in the backup and restore infrastructure, which might be interesting to avoid depending on your app’s needs.
+
+==== Device Encryption Detection (HAS_ENCRYPTION_ENABLED)
+IMPORTANT: This check is available for Android only
+
+To detect whether the devices filesystem is encrypted the `SecurityChecks#HAS_ENCRYPTION_ENABLED` function can be used.
+This function uses Android’s link:https://developer.xamarin.com/api/type/Android.App.Admin.DevicePolicyManager/[DevicePolicyManager] class.
+
+=== Custom Self Defence Checks
+
+These are self defence checks that you can define to be used by the SDK. Custom self defence checks must implement the link:javadoc[`ISecurityCheck`] interface.
```

Comment:
Maybe remove the link to the javadoc for now and create a github issue to remind us to add links once the javadocs are published?